src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message in the 'System.out.println' is not informative enough. It should indicate the specific type of machine, in this case, 'Truck', and what work it is doing.

Reply:
Fixed